virt/monitoring/virt-running-cluster-checkups.adoc

```diff
@@ -22,28 +25,28 @@ include::snippets/technology-preview.adoc[]
 
 include::modules/virt-about-cluster-checkup-framework.adoc[leveloffset=+1]
 
-== Running cluster checkups in the web console
+== Running checkups in the web console
 
-Use the web console to run a latency or storage checkup on a cluster.
-
-Use the following procedures the first time you run a latency checkup and storage checkup in the web console. For additional checkups, click *Run checkup* on either checkup tab, and select the appropriate checkup from the drop down menu.
-
-[IMPORTANT]
-====
-Before you run a latency checkup, you must first xref:../../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-connecting-vm-to-linux-bridge[create a bridge interface] on the cluster nodes to connect the VM's secondary interface to any interface on the node. If you do not create a bridge interface, the VMs will not start and the job will fail.
-====
+Use the following procedures the first time you run checkups in the web console. For additional checkups, click *Run checkup* on either checkup tab, and select the appropriate checkup from the drop down menu.
 
 include::modules/virt-latency-checkup-web-console.adoc[leveloffset=+2]
 
 include::modules/virt-storage-checkup-web-console.adoc[leveloffset=+2]
 
-include::modules/virt-measuring-latency-vm-secondary-network.adoc[leveloffset=+1]
 
-include::modules/virt-checking-cluster-dpdk-readiness.adoc[leveloffset=+1]
+== Running checkups on the CLI
```

Comment:
There also needs to be an ID above this heading:
```suggestion
[id="virt-running-cluster-checkups-cli_{context}"]
== Running checkups on the CLI
```

Reply:
Thanks here, too!